documentation/jetty-documentation/src/main/java/org/eclipse/jetty/docs/programming/server/http2/HTTP2ServerDocs.java

```diff
@@ -301,7 +302,7 @@ public void push() throws Exception
     {
         // tag::push[]
         // The favicon bytes.
-        ByteBuffer faviconBuffer = BufferUtil.toBuffer(ResourceFactory.root().newResource("/path/to/favicon.ico"), true);
+        ByteBuffer faviconBuffer = BufferUtil.toMappedBuffer(Paths.get("/path/to/favicon.ico"));
```

Comment:
Do not want to encourage use of mapped buffers in the docs.
Keep it simple and back to normal buffers.

Reply:
Reverted.